pkg/preparer/listener.go

```diff
@@ -124,6 +124,8 @@ func (l *HookListener) Sync(quit <-chan struct{}, errCh chan<- error) {
 				sub.WithField("err", err).Errorln("Could not write hook link")
 			} else {
 				sub.NoFields().Infoln("Updated hook")
+				// Send back a nil error to signal success!
+				errCh <- nil
```

Comment:
What kind of information are you thinking of including on such `events` on the `eventStream` channel?